src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -158,7 +149,6 @@ public ObjectNode getBrowserPreferences(BrowserType browserType) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes